src/rules/no_var.rs

```diff
@@ -26,6 +28,24 @@ impl LintRule for NoVar {
     let mut visitor = NoVarVisitor::new(context);
     visitor.visit_program(program, program);
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Enforces the use of block scoped variables over more error prone function scoped variables. Block scoped variables are defined using `const` and `let` keywords.
+
+`const` and `let` keywords ensure the variables defined using these keywords are not accessible outside their block scope. On the other hand, variables defined using `var` keyword are only limited by their function scope.
+
+### Invalid:
+```typescript
+var foo = 'bar';
```

Comment:
```suggestion
var foo = "bar";
```

It's trivial, but I think double quotations should be better, because `deno fmt` command, which is the built-in formatted of Deno, prefers them to single quotations.